responsibleai_vision/responsibleai_vision/utils/feature_extractors.py

```diff
@@ -81,10 +81,11 @@ def extract_features(image_dataset: pd.DataFrame,
                     # decode bytes
                     if isinstance(data, bytes):
                         data = data.decode()
-                    # TODO: add support for other data types
                     if isinstance(data, str):
                         feature_metadata.categorical_features.append(str(tag))
                         row_feature_values[feature_names.index(tag)] = data
+                    elif isinstance(data, int) or isinstance(data, float):
+                        row_feature_values[feature_names.index(tag)] = data
```

Comment:
how were you able to fix the json serialization error you were running into with the circular references?  or was that due to some other data type?

Reply:
I'm so glad you asked, I honestly dk how this suddenly got working loolll

I could try to guess rebuilding, but believe I had already done that before and ran into the same error.

Another hypothesis can be, I perhaps removed unwanted data / references by strictly pegging metadata to str, int, or float.